### ISSUE TITLE
Train with single GPU

### ISSUE THREAD

**Issue by 213pk** [soumik-kanad/diffssl#5]
How do I train the model with one GPU? Thanks!

**Comment by soumik-kanad** [comment]
In the training scripts change --nproc_per_node=4 to --nproc_per_node=1. This should run run with only 1 process (=1gpu). 

Remember to update the learning rate accordingly.
